src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/UpdateContactDetails.java

```diff
@@ -0,0 +1,150 @@
+package uk.gov.hmcts.divorce.caseworker.event.page;
+
+import uk.gov.hmcts.ccd.sdk.api.Event;
+import uk.gov.hmcts.ccd.sdk.api.FieldCollection;
+import uk.gov.hmcts.ccd.sdk.type.Organisation;
+import uk.gov.hmcts.ccd.sdk.type.OrganisationPolicy;
+import uk.gov.hmcts.divorce.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.divorce.ccd.PageBuilder;
+import uk.gov.hmcts.divorce.common.model.Applicant;
+import uk.gov.hmcts.divorce.common.model.Application;
+import uk.gov.hmcts.divorce.common.model.CaseData;
+import uk.gov.hmcts.divorce.common.model.MarriageDetails;
+import uk.gov.hmcts.divorce.common.model.Solicitor;
+import uk.gov.hmcts.divorce.common.model.State;
+import uk.gov.hmcts.divorce.common.model.UserRole;
+
+public class UpdateContactDetails  implements CcdPageConfiguration {
+    @Override
+    public void addTo(PageBuilder pageBuilder) {
+        FieldCollection.FieldCollectionBuilder<CaseData, State, Event.EventBuilder<CaseData, UserRole, State>> fieldCollectionBuilder
+            = pageBuilder.page("UpdateContactDetails");
+
+        FieldCollection.FieldCollectionBuilder<CaseData, State, Event.EventBuilder<CaseData, UserRole, State>> applicantFieldBuilder
+            = getApplicantFieldBuilder(fieldCollectionBuilder);
+        FieldCollection.FieldCollectionBuilder<CaseData, State, Event.EventBuilder<CaseData, UserRole, State>> respondentFieldBuilder
+            = getRespFieldBuilder(applicantFieldBuilder);
+
+        respondentFieldBuilder.label("Label-UpdateMarriageCertWarning",
+            "### Only update Marriage Certificate Names to make them match the marriage certificate exactly")
+            .complex(CaseData::getApplication)
+            .complex(Application::getMarriageDetails)
+            .optional(MarriageDetails::getApplicant1Name)
+            .optional(MarriageDetails::getApplicant2Name)
+            .done()
```

Comment:
I think you lost the formatting when you moved this out

Reply:
Formatted the code